sphinx_immaterial/local_mathjax.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+import shutil
+from typing import Dict, Any, cast
+from sphinx.application import Sphinx
+from sphinx.config import Config
+from sphinx.errors import ExtensionError
+from sphinx.ext.mathjax import MATHJAX_URL, MathDomain
+from sphinx.environment import BuildEnvironment
+
+
+def copy_mathjax_dist(app: Sphinx, env: BuildEnvironment) -> None:
+    if (
+        app.builder.format != "html"
+        or app.builder.math_renderer_name != "mathjax"  # type: ignore[attr-defined]
+    ):
+        return
+    if not app.config.mathjax_path:
+        raise ExtensionError(
+            "mathjax_path config value must be set for the mathjax extension to work"
+        )
+    if (
+        "mathjax_path" in app.config
+        and app.config["mathjax_path"] != "mathjax/tex-mml-chtml.js"
+    ):
+        return  # not using theme's cache of mathjax dist
+
+    domain = cast(MathDomain, env.get_domain("math"))
+    if app.registry.html_assets_policy == "always" or domain.has_equations():
+        # copy mathjax fonts only if equations exists
+        shutil.copytree(
+            str(Path(__file__).parent / "static" / "mathjax"),
```

Comment:
Since the `static` directory is by default always copied, it would be simpler to put mathjax and mermaid in a different directory and then there is no need to specifically exclude them.

Reply:
like "bundles" or were you thinking of a new folder?

Reply:
I think bundles is fine.